### DISCUSSION TITLE
Unable to boot Mac Classic from BlueSCSI

### DISCUSSION THREAD

**Discussion by mdwelsh** [erichelgeson/BlueSCSI-v1#30] [Q&A] [answered]
Hi there,

I have a Mac Classic with an internal hard drive that I would like to replace with a BlueSCSI. I bought the BlueSCSI assembled. I have formatted a 32GB SD card as ExFAT, and wrote the image HD00_512.hda to it. (I am not sure if this is the correct SCSI device ID.) I have the termination jumpers installed. 

When booting the machine comes up and shows a floppy disk icon with a blinking question mark. The PWR LED on the BlueSCSI is on continuously, and the lower LED blinks periodically 5 times. Any suggestions about how to diagnose this? I'm handy with hardware and Arduino if that helps. Thanks!

(I see from the code that this means the SD card was not found, and from another discussion forum post that reformatting the SD using the SD Card Formatter app may help - going to try that.)

**Comment by erichelgeson** [comment, marked as answer]
Yes - 5x means no sd card found. Try https://www.sdcard.org/downloads/formatter/ first and then try a different SD card if you have one. That 95% of the time will resolve the issues.

The file name looks correct.

**Reply by marius137** [reply]
the problem is probably that he formatted the card as ExFAT. AFAIK BlueSCSI Only supports FAT32

**Reply by erichelgeson** [reply]
@marius137 That is incorrect - Please take a look at the README in this repo - we recommend ExFAT as there is a performance gain when using that format.

**Comment by mdwelsh** [comment]
Okay, reformatting the SD using the SD Card Formatter app from https://www.sdcard.org/downloads/formatter/
solved the problem! Thanks!
